src/python/pants/backend/python/goals/pytest_runner.py

```diff
@@ -402,7 +402,7 @@ async def setup_pytest_for_target(
                 *(("-n", "{pants_concurrency}") if xdist_concurrency else ()),
                 *request.prepend_argv,
                 *pytest.args,
-                *coverage_args,
+                *pytest_args,
```

Comment:
This probably deserves an N.B. comment. Now that we're using command line options instead of the env var, it's critical pytest_args comes after pytest.args (which are specificed by the user via `[pytest] args`) just like you've done here. That should not change.

Reply:
(FYI `N.B.` means "nota bene", i.e. note well)